app/controllers/JavaListController.scala

```diff
@@ -66,6 +66,7 @@ class JavaListController @Inject()(versionRepo: VersionsRepository) extends Cont
   val vendors = Map(
     "adpt" -> "AdoptOpenJDK",
     "amzn" -> "Amazon",
+    "dwell" -> "Dragonwell",
```

Comment:
Can we use the vendor's name instead like the others? How about something like `albba` for Alibaba.